src/InMemoryRepository.php

```diff
@@ -81,7 +81,7 @@ public function find($query, $language = 'glob')
         $resources = array();
 
         if (Glob::isDynamic($query)) {
-            $resources = $this->getGlobIterator($query);
+            $resources = iterator_to_array($this->getGlobIterator($query));
```

Comment:
This isn't needed as the ArrayResourceCollection accepts iterators. You could add `Traversable` to the constructor argument's PHPDoc of that class instead.